spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/reload/ConfigReloadUtil.java

```diff
@@ -120,6 +121,10 @@ else if (propertySource instanceof MountConfigMapPropertySource mountConfigMapPr
 					// we know that the type is correct here
 					managedSources.add((S) mountConfigMapPropertySource);
 				}
+				else if (propertySource instanceof SecretsPropertySource secretsPropertySource) {
```

Comment:
I am fixing a bug here with this one. 

In case of configmaps, when we create a property source from paths, we create an instance of `MountConfigMapPropertySource`. Then, in `ConfigReloadUtil`, there is such a check:

```
if (source instanceof MountConfigMapPropertySource mountConfigMapPropertySource) { ... }
```

which means that reloads will take it into consideration.

---

While refactoring one integration test, I saw that it is failing, and after some debugging, realized that we have no code like the above for secrets. As such, I added that simple condition, because when we create a property source from paths in case of secrets, we create an instance of `SecretsPropertySource `